public/javascripts/songlistTable.js

```diff
@@ -2,52 +2,85 @@ var songlistTableComponent = (function() {
     var songID = m.prop()
     var inLibrary = m.prop(currentInLibrary)
     var playlistName = m.prop()
+    var displayedSongs = m.prop(songs)
+    var initial = 5
+    var langShown = languageFilterComponent.langShown
+
+    var loadMore = function(amount, langShown) {
+        m.request({
+                method: 'PUT',
+                url: '/',
+                data: {amount: amount, langShown: langShown}
+            })
+            .then(function(res) {
+                displayedSongs(res.songs)
+            })
+    }
 
     var songlistTable = {
         view: function() {
             return [
-                m('table.table', [
-                    m('thead', [
-                        m('th', 'Title'),
-                        m('th', 'Author'),
-                        m('th'),
-                        m('th', [
-                            m(playlistDropdownComponent.playlistDropdown, {
-                                playlistName: playlistName
+                m(languageFilterComponent.languageFilter, {songs: displayedSongs}),
+                m('.table-responsive', [
+                    m('table.table', [
+                        m('thead', [
+                            m('th', 'Title'),
+                            m('th', 'Author'),
+                            m('th'),
+                            m('th', [
+                                m(playlistDropdownComponent.playlistDropdown, {
+                                    playlistName: playlistName,
+                                    url: '/'
+                                })
+                            ])
+                        ]),
+                        m('tbody', [
+                            displayedSongs().map((s) => {
+                                s.label = s.label || m.prop('Add to Playlist');
```

Comment:
I see this code here also...as I mentioned earlier could the Add to Playlist text be passed into the component instead of hardcoded in all these places?